docs/source/conf.py

```diff
@@ -201,7 +201,12 @@
 
 # General information about the project.
 project = u'Apache Arrow'
-copyright = f'2016-{datetime.datetime.now().year} Apache Software Foundation'
+copyright = (
+    f"2016-{datetime.datetime.now().year} Apache Software Foundation.\n"
+    "Apache Arrow, Arrow, Apache, the Apache feather logo, and the Apache Arrow "
+    "project logo are either registered trademarks or trademarks of The Apache "
+    "Software Foundation in the United States and other countries."
```

Comment:
```suggestion
    "Software Foundation in the United States and other countries"
```
It seems there is already a dot printed by default.